[apm] clarify k8s settings for unified service tagging (#9251)
DataDog/documentation: content/en/getting_started/tagging/unified_service_tagging.md

### Containerized environment

> In containerized environments, `env`, `service`, and `version` are set through the service's environment variables or labels (e.g., Kubernetes deployment and pod labels, Docker container labels). The Datadog Agent can detect this tagging configuration and apply it on the data that it collects from containers.

Reviewer: ```suggestion
In containerized environments, `env`, `service`, and `version` are set through the service's environment variables or labels (for example, Kubernetes deployment and pod labels, Docker container labels). The Datadog Agent detects this tagging configuration and applies it to the data it collects from containers.
```
